### ISSUE TITLE
[@airbrake/browser] warning caused by incomplete esm sourcemaps

### ISSUE THREAD

**Issue by jlowcs** [airbrake/airbrake-js#916]
# 🐞 bug report

### Affected Package

`@airbrake/browser`

### Is this a regression?

Yes, I don't remember seeing these warnings in `1.0.6`.

### Description

Multiple warnings are appearing when compiling with webpack using the `esm` build.

Example:
![image](https://user-images.githubusercontent.com/3777724/106031567-b7053280-60cf-11eb-8cc2-2c057047bdd2.png)

This seems to be caused by the `"sources":["../../src/filter/angular_message.ts"]` and the lack of a `sourcesContent` in the map file.

I think changing the build config so that `sourcesContent` is included in the map files should remove the warning.

`sourcesContent` is present in the umd map files btw.

## 🔬 Minimal Reproduction

Build the lib, and look at the contents of the map files in the `esm` folder.

## 🔥 Exception or Error

See screenshot above

## 🌍  Your Environment

**@airbrake/\* version:**
<pre><code>
1.4.2
</code></pre>

**Comment by jlowcs** [comment]
I'm still experiencing this issue on 2.1.5. It's adding a lot of noise in the console and in the terminal when building with webpack.

**Comment by stianlp** [comment]
+1 Experiencing the same

**Comment by jboler** [comment]
I'm also seeing this with a React app using `create-react-app` v5.0.0

**Comment by sunpietro** [comment]
I confirm. It's still an issue with CRA v5.0.0

**Comment by martisj** [comment]
Experiencing this issue as well 😿

**Comment by ryanehrler** [comment]
> I'm also seeing this with a React app using `create-react-app` v5.0.0

Same here.  

I rebuilt the lib using `"inlineSources": true`, and published here - 

[airbrake-browser-esm](https://www.npmjs.com/package/airbrake-browser-esm)

**Comment by sunpietro** [comment]
@ryanehrler awesome. I'll check that

**Comment by kyrylo** [comment]
I am trying to reproduce this issue with `@airbrake/browser` v2.1.8 and `create-react-app` v5.0.1.

Here's what I did:

1. `npx create-react-app my-app`
2. Added the following line to `package.json`: `"type": "module",`
3. Added `@airbrake/browser`: `npm i --save @airbrake/browser`
4. Ran `npm run build` or `npm start`. Fixed import errors and tried again.
5. Everything worked and I didn't see any warnings

Can someone help me reproduce the problem?
